etc/DependencyInstaller.sh

```diff
@@ -34,6 +35,23 @@ _installCommon() {
     else
         pip3 install --no-cache-dir --user -U $pkgs
     fi
+
+    baseDir=$(mktemp -d /tmp/DependencyInstaller-orfs-XXXXXX)
+
+    # Install Verilator
```

Comment:
Nice. I think that we're asking for less trouble by building ourselves than to use the version that comes with the OS distribution.